myceli/src/shipper.rs

```diff
@@ -418,6 +418,7 @@ impl<T: Transport + Send + 'static> Shipper<T> {
             cid: Cid::try_from(block.cid)?.to_string(),
             data: block.data,
             links,
+            filename: block.filename,
```

Comment:
I may very well be misunderstanding the purpose of this mod, but are the various satellites & ground stations supposed to agree on the filenames of roots?

Reply:
The idea is to give recognizable names to DAGs/root CIDs for recognition by operators and/or observers.